docs/source/using/profiles.adoc

```diff
@@ -43,16 +64,17 @@ $ minishift profile list
 [[switching-profiles]]
 == Switching Profiles
 
-With the help of xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] you can switch between profiles or set a profile as active.
-When a profile is active, it is not required to pass `--profile` flag to run minishift commands against that profile.
+You can switch between profiles with the xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] command.
+Only one profile can be active at any time.
+You do not need to use the `--profile` flag to run {project} commands against the active profile.
```

Comment:
This sentence is not necessary here and distracts from what we are trying to say. Let's just remove it.